wire-ios/WireCommonComponents/Analytics/Datadog/WireDatadogBuilder.swift

```diff
@@ -27,28 +27,30 @@ struct WireDatadogBuilder {
     private enum Constants {
         static let keyAppId = "DatadogAppId"
         static let keyBundleVersion = "CFBundleVersion"
+        static let keyBundleShortVersion = "CFBundleShortVersionString"
         static let keyClientToken = "DatadogClientToken"
     }
 
     private let device: UIDevice = .current
     private let environment: BackendEnvironment = .shared
     private let bundle: Bundle = .wireCommonComponents
-    private let mainBundle: Bundle = .main
+    private let mainBundle: Bundle = .appMainBundle
 
     // MARK: - Build
 
     func build() -> WireDatadog {
         guard
             let applicationID = bundle.infoForKey(Constants.keyAppId),
-            // can buildNumber be also be refactored to use `infoForKey`?
-            let buildNumber = mainBundle.object(forInfoDictionaryKey: Constants.keyBundleVersion) as? String,
+            let buildNumber = mainBundle.infoForKey(Constants.keyBundleVersion),
+            let buildVersion = mainBundle.infoForKey(Constants.keyBundleShortVersion),
             let clientToken = bundle.infoForKey(Constants.keyClientToken)
         else {
             preconditionFailure("Datadog is enabled, but the bundle misses required input.")
         }
 
         return WireDatadog(
             applicationID: applicationID,
+            buildVersion: buildVersion,
```

Comment:
I had to reinject the application version in order to overwrite Datadog taking the bundle version of WireComponents (1).